docs/socialaccount/providers/saml.rst

```diff
@@ -165,3 +165,27 @@ The SAML provider has the following endpoints:
 - ``/accounts/saml/<organization_slug>/sls/``: Single Logout Service URL.
 
 - ``/accounts/saml/<organization_slug>/metadata/``: Metadata URL.
+
+Tips and Notes on SAML with Django:
+
+- Most SAML IdPs require TLS (nee SSL) to be used, making testing with
```

Comment:
What do you mean by "nee SSL" ?

Reply:
`née` is an adjective meaning "formerly known as", as I've found not everyone knows the TLS acronym. Perhaps I'll just change it to "formerly SSL" to clarify.

Reply:
"aka" is perhaps a more commonly used western English alternative to née.